### ISSUE TITLE
It doesn´t upgrade requirements 

### ISSUE THREAD

**Issue by VictorPelaez** [VictorPelaez/coral-reef-optimization-algorithm#25]
It doesn´t upgrade all requirements in the installation process. 
Example, numpy version is stacked in 1.12 and extremedepredation returns an error. Bug fixed in 1.13.1

Try to solve the upgrade when installed packages are bellow required version

**Comment by apastors** [comment]
Can you run this?
```python
In [1]: import numpy

In [2]: numpy.__version__
Out[2]: '1.13.1'
```

**Comment by VictorPelaez** [comment]
Yes, I have this numpy version, but I don't know why when a install in a friend's computer via pip, numpy version doesn't upgrade. With 1.12, numpy returns an error

**Comment by apastors** [comment]
Could he try installing a specific numpy version with pip?
```bash
pip install 'numpy==1.13.1'
```

Check [this thread](https://stackoverflow.com/questions/5226311/installing-specific-package-versions-with-pip) from SO

**Comment by apastors** [comment]
Shall I close this?
